pkg/Microsoft.Windows.Compatibility/test/publishcompatibilityassets.ps1

```diff
@@ -0,0 +1,60 @@
+$repoRoot = ((get-item $PSScriptRoot).parent.parent.parent.FullName);
+$winRID = "win-x64";
+$dotnetPath = -join($repoRoot, "\Tools\dotnetcli\dotnet.exe")
+$csprojPath = -join($PSScriptRoot, "\", (Get-ChildItem $PSScriptRoot"\*.csproj" | Select-Object -ExpandProperty Name))
```

Comment:
You don't need to pass the csproj really, given it is the only one in the directory it will just be discovered.

Reply:
If the script is ran from another folder, i.e the repo root, it would try to restore build.proj, that is why I passed the csproj, just to make sure that it always tries to restore and publish this csproj.